content/en/proposals/ha-nfd.md

```diff
@@ -0,0 +1,84 @@
+# Proposal-NNNN: Hardware Awareness scheduler based on NFD 

```

Comment:
We need to insert metadata at the beginning of this file. That's what the website theme requires.

```
---
title: "Hardware Awareness Scheduling Based on NFD"
description: "proposal for hardware awareness scheduling"
date: 2022-07-15
draft: false
weight: -2
---
```